src/utils/firebaseUtils.js

```diff
@@ -1,11 +1,25 @@
 import { db } from '../lib/firebase';
-import { collection, addDoc } from '@firebase/firestore';
+import { setDoc, doc, updateDoc, getDoc } from '@firebase/firestore';
 
-export const addProduct = ({ productName, timeFrame, lastPurchaseDate }) =>
-  addDoc(collection(db, 'products'), {
-    productName,
-    timeFrame,
-    lastPurchaseDate,
-    userToken: Math.floor(Math.random() * 11),
-    date: Date.now(),
+export const createListToken = (listToken) =>
+  setDoc(doc(db, 'listTokens', listToken), {});
```

Comment:
```suggestion
export const FIREBASE_COLLECTION_TOKENS_ID = "listTokens";
// ...
  setDoc(doc(db, FIREBASE_COLLECTION_TOKENS_ID, listToken), {});
```

If you create a variable, constant like FIREBASE_COLLECTION_TOKENS_ID, you can avoid "[magic values](https://kyleshevlin.com/what-are-magic-values/)" (Thanks Susana!), bugs, and improve the code maintainability. What is your opinion about it?

Reply:
or you can pass it as a parameter. It's another idea 😄 .